bitnami/redmine/Chart.lock

```diff
@@ -0,0 +1,12 @@
+dependencies:
+- name: common
+  repository: https://charts.bitnami.com/bitnami
+  version: 0.10.0
```

Comment:
Should be generated again with the new major of the bitnami/common subchart, but it is something the pipeline is going to update automatically when used this PR as a parameter in the Docker Image step